vale/styles/jargonLint/README.md

```diff
@@ -0,0 +1,9 @@
+# jargonList rules
+
+## Tooling
+
+substitution can be sorted with this
+
+```shell
+awk -FS=':' '{print $2 $1}' rule.yaml | sort | awk -FS=':' '{print $2 $1}'
```

Comment:
This code will be inaccessible if we separate it from the YAML file. Why not add it there with something like:
```
# You may sort the substitution with "awk -FS=':' '{print $2 $1}' rule.yaml | sort | awk -FS=':' '{print $2 $1}'"
```